rasa_sdk/interfaces.py

```diff
@@ -70,7 +70,7 @@ def __init__(
     @property
     def active_form(self) -> Dict[Text, Any]:
         warnings.warn(
```

Comment:
The linter check failed on line 71 because it expected a docstring, could you please add the following docstring:
```
Returns the active_loop, of type dictionary.
```